CMakeLists.txt

```
@@ -56,8 +56,16 @@ endif()
 
 add_compile_options(-Wall -Wextra -Werror -Wfatal-errors -Wno-missing-braces)
 
+if (NOT OTBR_VENDOR_NAME)
+    set(OTBR_VENDOR_NAME "OpenThread")
+endif()
```

same for product name.
```suggestion
set(OTBR_VENDOR_NAME "OpenThread" CACHE STRING "vendor name")
```